testUtilities/ConditionalCheckFailedException.ts

```diff
@@ -0,0 +1,11 @@
+export class ConditionalCheckFailedExceptionMock extends Error {
```

Comment:
This class is created solely for testing purposes. When we need to return `ConditionalCheckFailedException` in mocks, this class is used.